Docs/source/theory/kinetic_fluid_hybrid_model.rst

```diff
@@ -0,0 +1,141 @@
+.. _theory-kinetic-fluid-hybrid-model:
+
+Kinetic-fluid Hybrid Model
+==========================
+
+Many problems in plasma physics fall in a class where electron kinetics do not
+play a critical role in the solution. Examples of such situations include the
+study of collisionless magnetic reconnection and instabilities driven by ion
+temperature anisotropy, to mention only two. For these kinds of problems the
+computational cost of resolving the electron dynamics can be avoided by modeling
+the electrons as a fluid rather than kinetic particles. The simulation resolution
+can then be set by the ion time and length scales (commonly the ion cyclotron
+period :math:`1/\Omega_i` and ion skin depth :math:`l_i`, respectively), which
+can reduce the total simulation time drastically compared to a simulation that
+has to resolve the electron Debye length and CFL-condition based on the speed
+of light.
+
+Many authors have described variations of the kinetic ion & fluid electron model,
+generally refered to as particle-fluid hybrid or just hybrid-PIC models. The implementation
+in WarpX follows the outline from `Winske et al. (2022) <https://arxiv.org/abs/2204.01676>`_.
```

Comment:
Ideally, it would be great if you could use BibTeX, as started in #3787, when adding references in the theory section (in this file and all other relevant files). It should be possible to add the reference entries to the existing `refs.bib` file and just cite them wherever needed.

Reply:
This is awesome! Thanks for pointing it out. Quick question though: you mention switching to bibtex for references in the `theory` section. Should I not use bibtex for the references in the `Examples` section?

Reply:
I went ahead and changed the references in the `Examples` section to also use bibtex. We can undo if you don't like it.

Reply:
Thanks, I think it's good if we use it everywhere. I just mentioned the theory section because that's where references are typically used the most. Thank you for propagating the change to other sections of the docs, too.